mac

```diff
@@ -157,7 +157,14 @@ brew_install_or_upgrade 'openssl'
 brew unlink openssl && brew link openssl --force
 brew_install_or_upgrade 'libyaml'
 
-ruby_version="$(curl -sSL http://ruby.thoughtbot.com/latest)"
+find_latest_ruby() {
+  curl -sSL \
+    'https://raw.githubusercontent.com/postmodern/ruby-versions/master/ruby/versions.txt' \
+    | tail -n 2 \
```

Comment:
Do we know that a stable version will always be the last listing in that file? Will we start auto installing `2.4.0-preview1` (or whatever the next version is) once it's available? Is that a problem?